watson/utils.py

```diff
@@ -84,11 +85,15 @@ def _style_short_id(id):
         return fmt(element)
 
 
-def format_timedelta(delta):
+def format_timedelta(watson, delta):
```

Comment:
This is breaking the API, what about the following instead?

```suggestion
def format_timedelta(delta, as_decimal=False):
```

Reply:
That's true. I can change it to the way you suggested. Would you prefer if I made this a file-local variable in cli.py or do the call in every method that uses format_timedelta?

Reply:
I would prefer not having a separated method to format timedelta but a single entrypoint that does the heavy lifting. This should be sufficient for such a small piece of code and its testability. WDYT @k4nar?